components/datadog/updater/install_script.sh

```diff
@@ -111,6 +117,8 @@ elif [ "${OS}" = "RedHat" ]; then
     $sudo_cmd sh -c "echo -e '[datadog]\nname = Datadog, Inc.\nbaseurl = https://${yum_url}/${yum_repo_version}/${ARCH}/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\npriority=1\ngpgkey=${gpgkeys}' > /etc/yum.repos.d/datadog.repo"
     $sudo_cmd yum -y clean metadata
     $sudo_cmd yum -y install datadog-installer
+    $sudo_cmd DD_INSTALLER_REGISTRY="${DD_INSTALLER_REGISTRY}" DD_INSTALLER_BOOTSTRAP_VERSION="${DD_INSTALLER_BOOTSTRAP_VERSION}" DD_INSTALLER_REGISTRY_AUTH="${DD_INSTALLER_REGISTRY_AUTH}" /opt/datadog-installer/bin/installer/installer bootstrap
```

Comment:
Let's discuss before merging: I'm wondering if it's not better to keep this on the agent side especially with tests on the bootstrap process itself

Reply:
It should be here for individual VMs or the lab, but we could make it optional with a flag for the E2E.